src/components/appBar/CyVerseAppBar.js

```diff
@@ -112,71 +105,55 @@ function CyverseAppBar(props) {
     return (
         <React.Fragment>
             <div className={classes.root}>
-                <AppBar position="static" className={classes.appBar}>
+                <AppBar
+                    position="static"
+                    variant="outlined"
+                    className={classes.appBar}
+                >
                     <Toolbar>
                         <a
                             href={constants.CYVERSE_URL}
                             target="_blank"
                             rel="noopener noreferrer"
                         >
-                            <img
-                                src="/cyverse_whitelogo.png"
-                                alt="CyVerse"
-                            ></img>
+                            <img src="/de.png" alt="CyVerse"></img>
                         </a>
-                        <Typography
-                            className={classes.title}
-                            variant="subtitle2"
-                            noWrap
-                        >
-                            {formatHTMLMessage("discovery")}
-                            <br />
-                            {formatHTMLMessage("environment")}
-                        </Typography>
-
-                        <div
-                            className={classes.search}
-                            id={build(ids.APP_BAR_BASE_ID, ids.SEARCH)}
-                        >
+                        <div className={classes.search}>
                             <div className={classes.searchIcon}>
-                                <SearchIcon />
+                                <SearchIcon color="primary" />
                             </div>
                             <InputBase
-                                placeholder={formatMessage(intl, "search")}
+                                placeholder="Search…"
```

Comment:
Actually I rewrote most of the code in this page to get the new Look and Feel. This is more of a WIP now since, I still have to get it working with small screens.